src/main/twirl/gitbucket/core/main.scala.html

```diff
@@ -2,10 +2,10 @@
 @import gitbucket.core.plugin.PluginRegistry
 @import gitbucket.core.view.helpers
 <!DOCTYPE html>
-<html prefix="og: http://ogp.me/ns#">
+<html prefix="og: http://ogp.me/ns#" lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=5.0" />
```

Comment:
`[user-scalable="no"] is used in the <meta name="viewport"> element or the [maximum-scale] attribute is less than 5`
https://web.dev/meta-viewport/?utm_source=lighthouse&utm_medium=devtools